packages.bib

```diff
@@ -3,15 +3,16 @@ @Manual{R-base
   author = {{R Core Team}},
   organization = {R Foundation for Statistical Computing},
   address = {Vienna, Austria},
-  year = {2021},
+  year = {2022},
   url = {https://www.R-project.org/},
 }
 
 @Manual{R-bookdown,
   title = {bookdown: Authoring Books and Technical Documents with R Markdown},
   author = {Yihui Xie},
   year = {2022},
-  note = {https://github.com/rstudio/bookdown},
+  note = {R package version 0.28},
+  url = {https://CRAN.R-project.org/package=bookdown},
```

Comment:
Presumably we should update wherever this comes from to reflect the site is being made with quarto, not bookdown, and then this will vanish.